nrpy/infrastructures/BHaH/MoLtimestepping/tests/MoL_MoL_step_forward_in_time.c

```diff
@@ -15,26 +127,21 @@ void MoL_step_forward_in_time(commondata_struct *restrict commondata, griddata_s
     commondata->time = time_start + 0.00000000000000000e+00 * commondata->dt;
     // Set gridfunction aliases from gridfuncs struct
     // y_n gridfunctions
-    REAL MAYBE_UNUSED *restrict y_n_gfs = griddata[grid].gridfuncs.y_n_gfs;
+    MAYBE_UNUSED REAL *restrict y_n_gfs = griddata[grid].gridfuncs.y_n_gfs;
     // Temporary timelevel & AUXEVOL gridfunctions:
-    REAL MAYBE_UNUSED *restrict y_nplus1_running_total_gfs = griddata[grid].gridfuncs.y_nplus1_running_total_gfs;
-    REAL MAYBE_UNUSED *restrict k_odd_gfs = griddata[grid].gridfuncs.k_odd_gfs;
-    REAL MAYBE_UNUSED *restrict k_even_gfs = griddata[grid].gridfuncs.k_even_gfs;
-    REAL MAYBE_UNUSED *restrict auxevol_gfs = griddata[grid].gridfuncs.auxevol_gfs;
-    params_struct *restrict params = &griddata[grid].params;
-    REAL MAYBE_UNUSED *restrict xx[3];
+    MAYBE_UNUSED REAL *restrict y_nplus1_running_total_gfs = griddata[grid].gridfuncs.y_nplus1_running_total_gfs;
+    MAYBE_UNUSED REAL *restrict k_odd_gfs = griddata[grid].gridfuncs.k_odd_gfs;
+    MAYBE_UNUSED REAL *restrict k_even_gfs = griddata[grid].gridfuncs.k_even_gfs;
+    MAYBE_UNUSED REAL *restrict auxevol_gfs = griddata[grid].gridfuncs.auxevol_gfs;
+    MAYBE_UNUSED params_struct *restrict params = &griddata[grid].params;
+    MAYBE_UNUSED REAL *restrict xx[3];
     for (int ww = 0; ww < 3; ww++)
       xx[ww] = griddata[grid].xx[ww];
-    const int Nxx_plus_2NGHOSTS0 = griddata[grid].params.Nxx_plus_2NGHOSTS0;
-    const int Nxx_plus_2NGHOSTS1 = griddata[grid].params.Nxx_plus_2NGHOSTS1;
-    const int Nxx_plus_2NGHOSTS2 = griddata[grid].params.Nxx_plus_2NGHOSTS2;
+    MAYBE_UNUSED const int Nxx_plus_2NGHOSTS0 = griddata[grid].params.Nxx_plus_2NGHOSTS0;
+    MAYBE_UNUSED const int Nxx_plus_2NGHOSTS1 = griddata[grid].params.Nxx_plus_2NGHOSTS1;
+    MAYBE_UNUSED const int Nxx_plus_2NGHOSTS2 = griddata[grid].params.Nxx_plus_2NGHOSTS2;
     rhs_eval(Nxx, Nxx_plus_2NGHOSTS, dxx, y_n_gfs, k_odd_gfs);
```

Comment:
I'm not sure what is happening here. Nxx doesn't seem to be defined, or Nxx_plus_2NGHOSTS.